index.html

```diff
@@ -8,7 +8,6 @@
   <link rel="icon" type="image/x-icon" href="/favicon.ico">
   <link rel="preconnect" href="https://fonts.googleapis.com" />
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;700&display=swap" rel="stylesheet" />  
```

Comment:
on win as for me text becomes harder to read. also it looks smaller, so in some places text goes in one row instead of 2 as planned by design (looks not good).
![image](https://github.com/user-attachments/assets/49566b78-90e3-4334-b6f4-126563cd5b4c)

Reply:
At today's (09/09/2024) daily meeting it was decided to keep it as is and increase the font size as part of another task.